mcos/optimizer/base.py

```diff
@@ -2,7 +2,8 @@
 import numpy as np
 
 
-class Optimizer(ABC):
+class AbstractOptimizer(ABC):
```

Comment:
`AbstractOptimizer` makes it more clear that this is an Abstract Class. To me `Optimizer` seemed like it was the default optimizer.

Reply:
good idea